firmware/broadcom_trx.ksy

```diff
@@ -40,7 +40,15 @@ doc: |
 doc-ref:
   - https://github.com/openwrt/openwrt/blob/3f5619f/tools/firmware-utils/src/trx.c
   - https://web.archive.org/web/20190127154419/https://openwrt.org/docs/techref/header
+  - https://github.com/freebsd/freebsd-src/blob/373ffc62c158e52cde86a5b934ab4a51307f9f2e/sys/contrib/device-tree/Bindings/mtd/partitions/brcm%2Ctrx.txt
+  - https://github.com/ffainelli/firmware-tools/blob/90c8921bb24f71d534c4e519701399691641f681/untrx.c
+  - https://github.com/NetBSD/src/blob/05082e19134c05f2f4b6eca73223cdc6b5ab09bf/sys/dev/usb/if_bwfm_usb.c
+  - https://github.com/openbsd/src/blob/2207c4325726fdc5c4bcd0011af0fdf7d3dab137/sys/dev/usb/if_bwfm_usb.c
+  - https://android.googlesource.com/platform/hardware/broadcom/wlan/+/master/bcmdhd/dhdutil/include/trxhdr.h
+  - https://android.googlesource.com/platform/hardware/broadcom/wlan/+/master/bcmdhd/dhdutil/dhdu.c
   - https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/Documentation/devicetree/bindings/mtd/partitions/brcm,trx.txt
+  - https://github.com/torvalds/linux/blob/88b06399c9c766c283e070b022b5ceafa4f63f19/drivers/mtd/parsers/parser_trx.c
+  - https://github.com/torvalds/linux/blob/88b06399c9c766c283e070b022b5ceafa4f63f19/drivers/mtd/parsers/bcm47xxpart.c#L45
 
 instances:
   header:
```

Comment:
This instance is to defer parsing `header` until it's needed, or what? Why it isn't in `seq`?

Reply:
Splitted into multiple formats. One is for `broadcom_trx` header and another one is for `asus_trx` footer (any of them or both can be missing!). And this allowed to relicense `broadcom_trx` to ISC, since I have found Broadcom original sources under this license.